src/azul/es.py

```diff
@@ -0,0 +1,24 @@
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+from aws_requests_auth import boto_utils
+from aws_requests_auth.aws_auth import AWSRequestsAuth
+
+from azul import config
+
+
+def es_client():
+    host = config.es_endpoint[0]
```

Comment:
This could have been a tuple destructuring assignment:

`host, port = config.es_endpoint`